feder/institutions/views.py

```diff
@@ -123,7 +123,7 @@ def get_queryset(self):
         qs = Tag.objects.annotate(institution_count=Count("institution"))
         if self.q:
             qs = qs.filter(name__icontains=self.q)
-        return qs.order_by("-institution_count").all()
+        return qs.order_by("name").all()
```

Comment:
Czy jest to w ogóle konieczne?

Reply:
@ad-m Z tego, co widziałem, `institution_count` jest potem wyświetlane przy nazwie tagu w liście rozwijanej.

Reply:
Ale nie odnoszę się czy konieczna jest adnotacja, ale czy konieczne jest wprowadzenie tu sortowania po nazwie, skoro pisałeś, że ono jest domyślne.

Reply:
W sumie nie jest. Pozostawienie domyślnego sortowania daje ten sam efekt. Zrobiłem update.